server/src/Server.hpp

```
@@ -73,6 +61,11 @@ namespace server {
 
       void clearClientSlot(int player_id);
 
+      std::shared_ptr<ecs::ServerInputSystem> getInputSystem(uint32_t roomId) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)